### DISCUSSION TITLE
Does the S3 File Upload only work with the Budibase DB?

### DISCUSSION THREAD

**Discussion by oscargws** [Budibase/budibase#9680] [Help]
Hey team,

I'm attempting to use the S3 File Upload in a form thats connected to a Postgres Table, however I can not for the life of me get it to work. Is this because it only works for Budibase DB's? If not, does anyone have any step by step instructions for enabling S3 as a Data source, and then uploading a file in a form? As I've followed the steps in the docs and it doesn't appear to work. Cheers

I can see someone have a similar issue [here](https://github.com/Budibase/budibase/discussions/5481#discussioncomment-3508766)

**Comment by melohagan** [comment]
Hey @oscargws 

The S3 File Upload component will work independent of the form schema chosen. You just need to enter a custom name for the field name.
